### DISCUSSION TITLE
How to design this categories for text annotation?

### DISCUSSION THREAD

**Discussion by Roxanne0330** [inception-project/inception#4098] [Q&A]
Thank you, Reckart! Could you help me to design this annotation categories? I have tried but couldn't get the right design. Look forward to your reply! 
<img width="306" alt="Bildschirmfoto 2023-07-14 um 07 44 04" src="https://github.com/inception-project/inception/assets/139484750/c7a5b143-83ce-4769-a43d-0bae24faf1b4">
<img width="282" alt="Bildschirmfoto 2023-07-14 um 07 44 22" src="https://github.com/inception-project/inception/assets/139484750/8ac87baa-75f7-4b26-bfb9-0860b09c96a7">
<img width="281" alt="Bildschirmfoto 2023-07-14 um 07 44 29" src="https://github.com/inception-project/inception/assets/139484750/bb5ebe9e-a5bf-4ed9-b43f-a281af0e9462">

**Comment by reckart** [comment]
The system tries to make a good decision for the visualization based on your setting -   but these decisions may not be sensible in all cases - e.g. when you have a lot of features with tagsets. 

Go to the **Layers** panel in the project settings, select the layer/feature you want to edit and for those features where you currently see a choice list (e.g. `KAT-Rolle) change e.g. the **Editor type** property from **Auto** to **Combo-box** if you want to have a more compact layout.

You might also consider if it makes sense having all these features on a single layer (i.e. if they apply to the same kind of "thing"/text span) or if it may be more convenient to split some of them out to other layers.

For the `Kommentar` feature, I would recommend enabling the **Multiple rows** and **Show in tooltip popup** options in the feature details and disabling the **Curatable** and **Visible** options there.

**Reply by Roxanne0330** [reply]
Thank you, Reckart! I still have trouble with adding new features under one layer or should I put different categories like KAT 1 and KAT 2 under two different layers? Could you give me an example instruction an KAT 1? I don't know how to design.
<img width="1125" alt="Bildschirmfoto 2023-07-14 um 08 08 58" src="https://github.com/inception-project/inception/assets/139484750/e07d1584-d643-41fa-8ad5-9cf014b3d388">
<img width="307" alt="Bildschirmfoto 2023-07-14 um 08 11 49" src="https://github.com/inception-project/inception/assets/139484750/f3bf2bc0-4f8b-4c7b-881d-be2137a5b31d">

**Reply by reckart** [reply]
Typically, you would create a layer for a "kind of things" like "Part-of-Speech" or "Named Entity" or "Sentiment" or such. And then you would add the feature(s) of that thing. I would also have to get a good understanding of the domain you want to model and then possibly try out several ways of modeling it.
